comparison/benches/bench.rs

```diff
@@ -33,7 +34,16 @@ fn bench_everything(c: &mut Criterion) {
         .unwrap_or_else(rand::random);
     eprintln!("Using RANDOM_SEED={}", seed);
 
-    let data: Vec<_> = INPUT_SIZES.iter().map(|&l| Data::new(l, seed)).collect();
+    let data: Vec<_> = OFFSETS
+        .iter()
+        .flat_map(|&o| {
+            INPUT_SIZES
+                .iter()
+                .skip(o * 3)
```

Comment:
This is a bit magical; can it be `skip_while(|i| i <= o)`?

Reply:
I'll just remove it. The smaller buffer performs the same for unaligned and aligned inputs. That's the current implementation detail and is not obvious here.